.changeset/long-lions-do.md

```diff
@@ -0,0 +1,22 @@
+---
+'astro': minor
+---
+
+Adds a new property to the globals `Astro` and `APIContext` called `prerender`. The `prerender` represents whether or not the current page is prerendered:
```

Comment:
```suggestion
Adds a new property to the globals `Astro` and `APIContext` called `isPrerendered`. This boolean value represents whether or not the current page is prerendered.

The value returned will default to the project's default rendering strategy: `output: 'static'` will return `true` by default and `output: 'server'` will return `false` by default.

Otherwise, the value will be equal to the value of  `prerender` exported on pages that have explicitly opted in to or out of prerendering:
```
Maybe something like this?

Reply:
I agree here. `isPrerendered` or `prerendered` makes more sense to me.